server/storage/backend/backend.go

```diff
@@ -71,6 +73,9 @@ type Backend interface {
 
 	// SetTxPostLockInsideApplyHook sets a txPostLockInsideApplyHook.
 	SetTxPostLockInsideApplyHook(func())
+	// VerifyTxConsistency verifies data in ReadTx and BatchTx are consistent.
+	VerifyTxConsistency(bucket Bucket)
+	UnsafeVerifyTxConsistency(bucket Bucket)
```

Comment:
Let's try not change the important interface just for the test. Can you find an alternative way to do it?

Reply:
changed it to a verify util function.